Modules/@babylonjs/react-native/ios/BabylonNative.cpp

```diff
@@ -72,6 +75,7 @@ namespace Babylon
         m_impl->m_graphics->AddToJavaScript(m_impl->env);
 
         Polyfills::Window::Initialize(m_impl->env);
+        Polyfills::XMLHttpRequest::Initialize(m_impl->env);
```

Comment:
Maybe a comment here indicating why we have a separate XMLHttpRequest on top of RN's?